rg_instructor_analytics/tasks.py

```diff
@@ -23,3 +36,90 @@ def send_email_to_cohort(subject, message, students):
     msg.encoding = 'UTF-8'
     msg.attach_alternative(html_content, "text/html")
     msg.send(fail_silently=False)
+
+
+cron_settings = getattr(
+    settings, 'RG_ANALYTICS_ENROLLMENT_STAT_UPDATE',
+    {
+        'minute': '*/1',
+        'hour': '*',
```

Comment:
I would leave the only `minute` key as default value

Reply:
done